.circleci/config.yml

```
@@ -237,6 +251,12 @@ commands:
                 command: |
                   brew install curl
                   echo 'export PATH="/usr/local/opt/curl/bin:$PATH"' >> $BASH_ENV
+            - run:
+                name: Install cmake
+                # TODO: If anyone understands brew better than me, please find a way to pin a specific version of cmake,
+                # that matches << pipeline.parameters.cmake_version >> thanks!
+                command: |
+                  brew install cmake
```

not sure if this works but maybe worth a shot
```suggestion
                  brew install cmake@<< pipeline.parameters.cmake_version >>
```

---

I tried 😭